lib/helpbase.gi

```diff
@@ -978,6 +978,21 @@ InstallGlobalFunction(HELP_GET_MATCHES, function( books, topic, frombegin )
   # Note: before GAP 4.5 this was only done in case of substring search.
   match := Concatenation(exact, match);
   exact := [];
+  # check if all matches point to the same subsection of the same book
```

Comment:
This comment says what is done, but not why. Perhaps add: "; if so, only keep the first match so that the help viewer can immediately show that help section, instead of asking the user to select one of the (equivalent!) matches"